zebra-utils/src/bin/zebra-checkpoints/main.rs

```diff
@@ -1,32 +1,108 @@
+//! Prints Zebra checkpoints as "height hash" output lines.
+//!
+//! Get all the blocks up to network current tip and print the ones that are
+//! checkpoints according to rules.
+//!
+//! Usage: zebra-checkpoints [network] --cli <cli-path>
+//! Where network can be `--regtest`, `mainnet` or `--testnet`.
+//! `--cli` is the path of the zcash-cli binary as a string.
+//! If no network is specified program will use the mainnet.
```

Comment:
We should use `zcash-cli` in the user's $PATH by default, rather than requiring that the user to specify a binary name or path.

(But we should definitely allow the user to specify the name or path of `zcash-cli`.)

Reply:
Added a default value at https://github.com/ZcashFoundation/zebra/pull/740/commits/d2e8257792c934f14b9e2688928b62e24cf934f0